Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -370,13 +376,23 @@ void Start()
 
             gameObject.SetActive(true); // client disables extra cameras, so set it true
             transform.position = new Vector3(0f, 10000000f, 0f); // keep this in a far away place
-
+            initialUpdateMode = audioRenderTexture.updateMode;
 
             // Disable camera on start if user didn't ask for it
             if (!audioDataToggle)
             {
                 DisableReadback();
             }
+
+#if !UDONSHARP && !CVR_CCK_EXISTS
+            Invoke(nameof(AutoCacheAudioTarget), 1);
```

Comment:
Why the Invoke delay?

Reply:
Related to `AutoCacheAudioTarget`. See https://github.com/llealloo/audiolink/pull/341/files#r2193355775